### DISCUSSION TITLE
2023, 2024 versions

### DISCUSSION THREAD

**Discussion by DunxUK** [kudrykv/latex-yearly-planner#20] [General]
Hi there! This is perfect for me but I work often years ahead, would it be possible to build future versions of rm2.mos.lined.default.ampm.dailycal.2022.pdf?

Also is there a way I can contribute some money, I am so grateful for your fantastic work!

Best regards

**Comment by kudrykv** [comment]
Here you go—
[rm2.mos.lined.default.ampm.dailycal.2023.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7793124/rm2.mos.lined.default.ampm.dailycal.2023.pdf)
[rm2.mos.lined.default.ampm.dailycal.2024.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7793126/rm2.mos.lined.default.ampm.dailycal.2024.pdf)

You can try with this link: https://send.monobank.ua/33UmjsvZD7 (name / card number / details / amount / comment)
currency there is UAH; e.g., 150 UAH ~ $5.50
(it likely wouldn't work; I am still figuring this out)

**Reply by DunxUK** [reply]
Done!

**Reply by gotchai** [reply]
> Here you go— [rm2.mos.lined.default.ampm.dailycal.2023.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7793124/rm2.mos.lined.default.ampm.dailycal.2023.pdf) [rm2.mos.lined.default.ampm.dailycal.2024.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7793126/rm2.mos.lined.default.ampm.dailycal.2024.pdf)
> 
> You can try with this link: https://send.monobank.ua/33UmjsvZD7 (name / card number / details / amount / comment) currency there is UAH; e.g., 150 UAH ~ $5.50 (it likely wouldn't work; I am still figuring this out)

Hi
Could you please issue a mos lined default (without am pm) version for 2023 and 2024 as well ?

Regards
Thank you for you wonderful work

**Reply by kudrykv** [reply]
@gotchai sure -- in a few days though.

**Reply by kudrykv** [reply]
@gotchai 
[rm2.mos.lined.default.dailycal.2023.pdf](https://github.com/kudrykv/latex-yearly-planner/files/8233679/rm2.mos.lined.default.dailycal.2023.pdf)
[rm2.mos.lined.default.dailycal.2024.pdf](https://github.com/kudrykv/latex-yearly-planner/files/8233680/rm2.mos.lined.default.dailycal.2024.pdf)

**Reply by gotchai** [reply]
> @gotchai [rm2.mos.lined.default.dailycal.2023.pdf](https://github.com/kudrykv/latex-yearly-planner/files/8233679/rm2.mos.lined.default.dailycal.2023.pdf) [rm2.mos.lined.default.dailycal.2024.pdf](https://github.com/kudrykv/latex-yearly-planner/files/8233680/rm2.mos.lined.default.dailycal.2024.pdf)

Thank you very much ! Very good job !

**Reply by DunxUK** [reply]
Believe it or not looking for 2025 and 2026 in the same format as before! Happy to send more coffee 🙂

Best regards

Duncan

**Reply by kudrykv** [reply]
Hi @DunxUK!
I'll generate them sometime later today or tomorrow

**Reply by kudrykv** [reply]
@DunxUK 
[rm2.mos.lined.default.ampm.dailycal.2025.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9893154/rm2.mos.lined.default.ampm.dailycal.2025.pdf)
[rm2.mos.lined.default.ampm.dailycal.2026.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9893155/rm2.mos.lined.default.ampm.dailycal.2026.pdf)

**Reply by breadelbane** [reply]
> Here you go— [rm2.mos.lined.default.ampm.dailycal.2023.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7793124/rm2.mos.lined.default.ampm.dailycal.2023.pdf) [rm2.mos.lined.default.ampm.dailycal.2024.pdf](https://github.com/kudrykv/latex-yearly-planner/files/7793126/rm2.mos.lined.default.ampm.dailycal.2024.pdf)
> 
> You can try with this link: https://send.monobank.ua/33UmjsvZD7 (name / card number / details / amount / comment) currency there is UAH; e.g., 150 UAH ~ $5.50 (it likely wouldn't work; I am still figuring this out)

Love this, do you have a left-handed Monday start for 2023. I found the 2024 version.

**Reply by kudrykv** [reply]
@breadelbane please try
[rm2_ddvk_lh.mos.lined.default.ampm.dailycal.2023.pdf](https://github.com/kudrykv/latex-yearly-planner/files/10226280/rm2_ddvk_lh.mos.lined.default.ampm.dailycal.2023.pdf)

**Reply by DunxUK** [reply]
Hey there!

If you would like some more coffee I could now really use 2027 and 2028 in the same rrm2.mos.lined.default.ampm.dailycal.2022.pdf format please? :) Is the link for sending money still the same? Many thanks once again!

**Reply by kudrykv** [reply]
Hi!
I'm glad you like this planner 🙂
The link is the same

[rm2.mos.lined.default.ampm.dailycal.2027.pdf](https://github.com/kudrykv/latex-yearly-planner/files/12701563/rm2.mos.lined.default.ampm.dailycal.2027.pdf)
[rm2.mos.lined.default.ampm.dailycal.2028.pdf](https://github.com/kudrykv/latex-yearly-planner/files/12701564/rm2.mos.lined.default.ampm.dailycal.2028.pdf)

**Comment by DunxUK** [comment]
I would love to try to do this but need help! What would the yaml files be for this type?

**Reply by kudrykv** [reply]
Do you mean how you can build it yourself?

**Reply by kudrykv** [reply]
If you're comfortable with shell, check out `release.sh` and edit it. Comment everything you don't need — there are configurations and permutations to comment out. Then, put years you want to generate and run it.

Otherwise I can make those edits for you, but you'll still need ~latest Go and LaTeX in your system installed.
